packages/common/package.json

```diff
@@ -72,5 +72,17 @@
     "@types/node": "^18.15.11",
     "tsup": "^6.7.0",
     "vitest": "0.34.6"
+  },
+  "peerDependencies": {
+    "@aws-sdk/client-kms": "^3.556.0",
+    "asn1.js": "^5.4.1"
```

Comment:
I _think_ we can be more flexible with the version ranges here

Reply:
Why would we want to be flexible?

Reply:
if someone wanted to use `common` for the kms account but there was a bug in a particular version of either of the packages above, I think pnpm would keep you from installing other versions because they wouldn't satisfy the above semver

Reply:
for peer deps its common to use a wide semver (like `3.x` instead of `^3.556.0`) or just use `*`, but with the expectation that calls to our lib may break if there were breaking changes in the dep version you use

Reply:
Done!